pkg/scrape/scrape.go

```diff
@@ -216,9 +216,9 @@ func (sp *scrapePool) Sync(tgs []*targetgroup.Group) {
 		}
 
 		for _, t := range targets {
-			if t.Labels().Len() > 0 {
+			if !t.Labels().IsEmpty() {
```

Comment:
For reference: https://github.com/prometheus/prometheus/blob/2d588725c12e46e4e71da3899ec05bb933151021/model/labels/labels.go#L421